### ISSUE TITLE
None in maps

### ISSUE THREAD

**Issue by yardenmatok203** [calico/basenji#171]
Hey,

Is it possible to have nan in input\prediction?

Also, there are regions in the genome that are problematic, did you ignore them?

Thanks,
Yarden

**Comment by davek44** [comment]
NaNs indicate that something has gone wrong. Yes, the training data creation stage uses a blacklist of regions that frequently encounter problematic read mapping.

**Comment by yardenmatok203** [comment]
where can I find this blacklist?

also, I see there is a variable "seq_hic_nan", so can it be that some of the values are nan?

last question, I have "allValidPairs" format, how can I create cool in your binsize\format?


Thanks, 
Yarden

**Comment by davek44** [comment]
Yes, for HiC NaNs regularly occur, and we interpolate to set the values.

I'm not familiar with allValidPairs format.

**Comment by yardenmatok203** [comment]
Do you know which library can I use?
In my HIC data I have data with bins: 500, 1000, 1500,....

Thank you,
Yarden
